test/parallel/test-event-emitter-symbols.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+require('common');
+const EventEmitter = require('events');
+const assert = require('assert');
+
+const ee = new EventEmitter();
+const foo = Symbol('foo');
+let didRun = false;
+
+function listener() {
+  didRun = true;
+}
+
+ee.on(foo, listener);
+assert.deepEqual(ee.listeners(foo), [listener]);
+
+ee.emit(foo);
+assert(didRun);
+
+ee.removeAllListeners();
+assert.deepEqual(ee.listeners(foo), []);
+
+ee.on(foo, listener);
+ee.removeListener(foo, listener);
```

Comment:
Let me try to explain better. You called `removeAllListeners()`, leaving an empty array of listeners on line 22. Then, on line 26, you're asserting that the array is empty. I just meant it might be worthwhile to add an assertion after line 24 that the array was no longer empty.